qiita_ware/search.py

```diff
@@ -0,0 +1,59 @@
+from collections import Counter
+
+from future.utils import viewvalues
+
+from qiita_db.search import QiitaStudySearch
+
+
+def count_metadata(results, meta_cols):
```

Comment:
It is counting the number of occurrences of each metadata value for each metadata colun. It's there specifically for qiita_pet.